cpu/samd21/Kconfig

```diff
@@ -5,4 +5,51 @@
 # directory for more details.
 #
 
+config CPU_FAMILY_SAMD2X
+    bool
+    select CPU_CORE_CORTEX_M0PLUS
+    select CPU_COMMON_SAM0
+
+config CPU_SERIES_SAMD21
+    bool
+    select CPU_FAMILY_SAMD2X
+    select HAS_CPU_SAMD21
+    select HAS_PUF_SRAM
+
+## CPU Models
+config CPU_MODEL_SAMD21G18A
+    bool
+    select CPU_SERIES_SAMD21
+
+config CPU_MODEL_SAMD21J18A
+    bool
+    select CPU_SERIES_SAMD21
+
+config CPU_MODEL_SAMR21E18A
```

Comment:
SAMD21E18A is missing (required by `serpente`)

Reply:
Done